### DISCUSSION TITLE
The building process in the pull requests always shows "Expected - Waiting for status to be reported", what does it mean?

### DISCUSSION THREAD

**Discussion by lxliyou001** [apache/doris#30889] [A - General / Q&A]
doris pull requests 中build流程一直显示 "Expected — Waiting for status to be reported"是什么意思 求助

**Comment by yagagagaga** [comment]
Don't worry, the only thing you need to do is to be patient.

**Comment by JNSimba** [comment]
You can comment `run buildall`
